infrastructure/web/lesson_video_buckets.tf

```diff
@@ -0,0 +1,23 @@
+// I want to create a bucket in gcp to hold lesson videos
+resource "google_storage_bucket" "lesson_videos" {
+  name          = "lesson-videos-${var.environment}"
+  location      = var.region
+  force_destroy = var.environment != "production"
+
+  versioning {
+    enabled = false
+  }
+
+  cors {
+    origin          = [var.retool_origin]
+    method          = ["GET", "HEAD", "PUT", "POST", "DELETE"]
```

Comment:
Is there a reason for specifying the lst 3 methods?

If this bucket is only intended to be read over http then I think we should only specify `["GET", "HEAD"]`

Reply:
Delete and one more are the two that arent necessary. 
Retool will be uploading the videos and displaying the videos but I need to investigate further to find out if it is POST or PUT it uses.